pom.xml

```diff
@@ -368,6 +368,11 @@
                 <artifactId>value-annotations</artifactId>
                 <version>2.9.3</version>
             </dependency>
+            <dependency>
+                <groupId>org.jooq</groupId>
+                <artifactId>jooq</artifactId>
+                <version>3.16.14</version>
```

Comment:
This is the latest version of jOOQ supporting Java 11. From 3.17.x, it is not supported.